rich/_inspect.py

```diff
@@ -233,3 +233,29 @@ def _get_formatted_doc(self, object_: Any) -> Optional[str]:
         if not self.help:
             docs = _first_paragraph(docs)
         return escape_control_codes(docs)
+
+
+def object_types_tree(obj: Union[object, Type[Any]]) -> Tuple[type, ...]:
```

Comment:
doc strings por favor

This doesn't seem to return a tree per se. Could it be `get_mro` ?